charts/portal/values-production.yaml

```diff
@@ -72,6 +72,13 @@ tls:
 # If you're migrating from a docker swarm environment and wish to keep your old internal certificates
 # then follow the instructions here to load your certificates into secrets manually #### Link to doc.
```

Comment:
# If you're migrating from a docker swarm environment and wish to keep your old internal certificates
# then follow the instructions here to load your certificates manually - https://github.com/CAAPIM/apim-charts/blob/stable/utils/portal-migration/README.md

Reply:
updated